src/__tests__/secretenv.integration.test.ts

```diff
@@ -78,6 +99,33 @@ describe('ResolveEnv', () => {
       console.error('error creating test AWS SSM parameter (unencrypted)', e);
       throw e;
     }
+    // Create AWS secret with multiple versions
+    /** AWSCURRENT */
+    try {
+      await secretsClient
+        .createSecret({
+          Name: testAwsSecretsManagerTestSecretName,
+          SecretString: testAwsSecretsManagerTestSecretCurrentValue,
+        } as AWS.SecretsManager.CreateSecretRequest)
+        .promise()
+        .then((res: AWS.SecretsManager.CreateSecretResponse) => {
+          testAwsSecretsManagerTestSecretArn = res.ARN ?? '';
+          testAwsSecretsManagerTestSecretCurrentId = res.VersionId;
```

Comment:
I pulled this down to try to get some context but I don't see anywhere else it is used. Is the CurrentId supposed to be used for other tests in the future or is there something I am missing?

Reply:
No I guess I didn't really end up using it for anything.